docs/en/guides/developer/transactional.md

```diff
@@ -0,0 +1,50 @@
+# Transactional (ACID) support
+
+INSERTs into one partition in one table of the MergeTree\* family up to `max_insert_block_size` rows are transactional (ACID):
+
+### Atomic
+
+INSERTs succeed or are rejected as a whole- if a confirmation is sent to the client, then all rows were inserted; if an error is sent to the client, then no rows were inserted.
+
+### Consistent
+
+If an INSERT succeeded, then all rows were inserted and they do not violate table constraints; if constraints were violated, then no rows were inserted.
+
+### Isolated
+
+Clients observe a consistent snapshot of the table - either the state of the table before an INSERT, or the state after a successful INSERT; no partial state is seen.
+
+### Durable
+
+A successful INSERT is written to the filesystem before sending a confirmation to the client, on single replica or multiple replica systems (this is controlled by the `insert_quorum` setting), and ClickHouse can ask the operating system to sync the filesystem data on the storage media (this is controlled by the `fsync_after_insert` setting).
+
+### Details
```

Comment:
I would not add it on the same level as ACID headers. Probably converting previous headers to a table or just a list will make it less tricky.

Reply:
see above screen-grab